From 5916d9910f65ea9f9796a62917b42b05a4e570d9 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Thu, 13 May 2010 19:27:15 -0400
Subject: [PATCH 221/391] ltt-relay-locked-api-update

ltt relay lockled api update

update to new reader API from
ltt-relay-alloc-use-array-and-vmalloc_sync_all.patch

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
---
 ltt/ltt-relay-locked.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ltt/ltt-relay-locked.c b/ltt/ltt-relay-locked.c
index 0f83c14..2c2e389 100644
--- a/ltt/ltt-relay-locked.c
+++ b/ltt/ltt-relay-locked.c
@@ -634,7 +634,7 @@ static int subbuf_splice_actor(struct file *in,
 
 	for (; spd.nr_pages < nr_pages; spd.nr_pages++) {
 		unsigned int this_len;
-		struct buf_page *page;
+		struct page *page;
 
 		if (!len)
 			break;
@@ -643,7 +643,7 @@ static int subbuf_splice_actor(struct file *in,
 
 		this_len = PAGE_SIZE - poff;
 		page = ltt_relay_read_get_page(buf, roffset);
-		spd.pages[spd.nr_pages] = page->page;
+		spd.pages[spd.nr_pages] = page;
 		spd.partial[spd.nr_pages].offset = poff;
 		spd.partial[spd.nr_pages].len = this_len;
 
-- 
1.6.5.2

